daemons/fenced/fenced_commands.c

```diff
@@ -1124,7 +1124,8 @@ dynamic_list_search_cb(GPid pid, int rc, const char *output, gpointer user_data)
         if (!alias) {
             alias = search->host;
         }
-        if (pcmk__str_in_list(dev->targets, alias, pcmk__str_casei)) {
+        if (pcmk__str_in_list(dev->targets, alias,
+                              pcmk__str_star_matches|pcmk__str_casei)) {
```

Comment:
We don't need star_matches in fenced_commands.c, pacemaker-fenced.c, or st_client.c (not that a node can be named \*, but it could be confusing)

Reply:
Updated.  The original patches were just trying to be as safe as possible on this stuff.